features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -71,9 +77,10 @@ internal class SessionReplayFeature(
         }
     )
 
-    internal lateinit var appContext: Context
+    private lateinit var appContext: Context
     private var isRecording = AtomicBoolean(false)
     internal var sessionReplayRecorder: Recorder = NoOpRecorder()
+    internal var resourcesWriter: ResourcesWriter = NoOpResourcesWriter()
```

Comment:
if this one temporary (until we use the one from the feature), shouldn't it be `val`?